server/src/testFixtures/java/io/crate/integrationtests/SQLIntegrationTestCase.java

```diff
@@ -323,9 +325,16 @@ public void assertNoTasksAreLeftOpen() throws Exception {
             String[] nodeNames = internalCluster().getNodeNames();
             for (String nodeName : nodeNames) {
                 TasksService tasksService = internalCluster().getInstance(TasksService.class, nodeName);
+                JobsLogService jobsLogService = internalCluster().getInstance(JobsLogService.class, nodeName);
```

Comment:
can we take it out of the loop?

Reply:
It was using the nodeName of the loop. But your comment made me realize the task may not be on the same node as the jobsLog entry so I changed it to just print out all job log entries across all nodes. In a regular run those should all be empty at this point - so anything left in their is useful information.